routes/students.routes.js

```diff
@@ -0,0 +1,83 @@
+const router = require("express").Router()
+
+const User = require("../models/User.model")
+
+const {isLogged, isPM} = require("../utils")
+
+const { isLoggedIn, checkRoles } = require("../middleware")
+
+
+//Students List
+router.get('/', (req, res) => {
+    
+    User
+    .find()
+    .then(regUser => {
+        res.render('students/students', { regUser, isLogged: isLogged(req.session.currentUser), isPM: isPM(req.session.currentUser?.role) })
+    })
+    .catch(err => console.log(err))
+})
```

Comment:
Ahora mismo la app te muestra todos los usuarios no solo los estudiantes.

```suggestion
router.get('/', (req, res) => {
    
    User
    .find({role: "Student"})
    .then(regUser => {
        res.render('students/students', { regUser, isLogged: isLogged(req.session.currentUser), isPM: isPM(req.session.currentUser?.role) })
    })
    .catch(err => console.log(err))
})
```